src/collections/_documentation/performance/alerts-notifications/alerts.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Alerts
+sidebar_order: 1
+---
+
+Alerts can be created by users with admin permission or higher from **Project Settings > Alerts > New Alert**.
+
+There are two types of alerts: Issue Alerts and Metric Alerts. 
+
+## Metric Alerts
+
+Metric alerts allow you to filter and set thresholds on all errors across a project. They can be used for high-level monitoring of patterns, or fine-grained monitoring of individual events. Metric alerts help express lightweight service-level objectives (SLOs) such as users affected by signup-page errors, or volume of database errors.
+
+### Alert Builder
+
+[ builder image ]
+
+A metric alert can have at most two triggers. The first one is a (required) critical trigger, and the second one is an (optional) warning trigger. Triggers are independent with one constraint: the warning trigger must be reached before the critical one.
```

Comment:
Suggest: "A metric alert has, at most, two triggers. The first is a critical trigger, which is required. The second is a warning trigger, which is optional. Triggers are independent of one another; however, the warning (optional) must be reached before the critical (required) trigger."